Content for requests to Safe ecosystem foundation.md

```diff
@@ -0,0 +1,70 @@
+### Forms
+
+**Form on https://safe.global/**
+
+Contact us
+https://notionforms.io/forms/get-in-touch-with-the-safe-ecosystem-foundation
+
+Submit your project (for technical integrations only)
+https://github.com/safe-global/safe-ecosystem-database/issues/new?assignees=&labels=project+%3Agame_die%3A&template=suggest_project.yaml&title=Suggest+a+project
+
+**Forms on https://www.gnosis.builders/contact-us**
+
+Collaborate with us
+https://forms.monday.com/forms/8636cfde490a3ce083f083e5376487a9?r=use1
+
+Connect with the team 
+https://forms.monday.com/forms/37fe262572726e8f752319bea04448bd?r=use1
+
+---
+
+### 1. Safe to provide bounties for DevPool
+
+**COMPANY/DAO/PROJECT NAME**
+
+DevPool by Ubiquity DAO
+
+**COMPANY/PROJECT WEBSITE** 
+
+https://github.com/ubiquity/devpool-directory
```

Comment:
```suggestion
https://github.com/ubiquity/devpool-directory/issues
```

Reply:
@pavlovcik I accepted this suggestion but then realised that this point is about COMPANY/PROJECT WEBSITE and I reverted the link.

The link to DevPool issues is provided as well in other point - PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES ...
https://github.com/ubiquity/devpool-directory/issues

Maybe change the link in the point COMPANY/PROJECT WEBSITE to https://dao.ubq.fi/devpool ?